src/features/unified-share-modal/SharedLinkSection.js

```diff
@@ -457,6 +458,7 @@ class SharedLinkSection extends React.Component<Props, State> {
         const canAddSharedLink = this.canAddSharedLink(isSharedLinkEnabled, item.grantedPermissions.itemShare);
         const canRemoveSharedLink = this.canRemoveSharedLink(isSharedLinkEnabled, canChangeAccessLevel);
         const isToggleEnabled = (canAddSharedLink || canRemoveSharedLink) && !submitting;
+        const toggleDescriptionId = uniqueId('toggle-description');
```

Comment:
I'm worried that this will create a new unique ID on every render. Any chance we can use something like:
`const [toggleDescriptionId] = React.useState(uniqueId('toggle-description'))`

Reply:
This is not a functional component, i referenced other examples and moved it to the top!